src/components/PrintButton/PrintButton.js

```diff
@@ -17,6 +17,7 @@ class PrintButton extends React.Component {
     dataSource: PropTypes.object,
     template: PropTypes.string,
     onBeforePrint: PropTypes.func,
+    onAfterPrint: PropTypes.func,
```

Comment:
If this property required you should mark it as isRequired in other case it should be listed in defautProps with default value

Reply:
Add onAfterPrint func into defaultProps

Reply:
In this case use can use lodash noop instead of arrow function. But are you sure that the prop is optional?